src/components/Lesson/ImprovePage.scss

```diff
@@ -0,0 +1,76 @@
+@import '../../styles/variables';
+
+.link {
+  text-decoration: none !important;
+  text-align: center;
+  font-size: 16px;
```

Comment:
I utgangspunktet er det best om font-size settes med enheten "em", der 1em bare betyr samme font-størrelse som parent, og e.g. "1.5em" er 50% større enn parent. Font-størrelsen til <html> er satt til $font-size-base (for tiden definert i src/styles/pre-customizations.scss til 16px, og satt i src/styles/_general.scss; forøvrig bør kommentaren i _general.scss leses!). Grunnen til å bruke em, er at dersom man senere bestemmer seg for å endre font-size-base, så vil alle andre fonter endre seg i takt (med mindre man har satt px). Det gjelder også for parent font-størrelser som endres av media-queries, og det er jo stort sett fint at størrelsene er riktig i forhold til hverandre.

Det finnes så klart unntak, dersom man vil at en font skal ha en bestemt størrelse uansett hva fontene rundt har som størrelse, men det bør være litt spesielt. Det eneste unntaket jeg kommer på i farten er størrelsen i breadcrumb, fordi den er ment å være 50px høy, og vi vil ikke ødelegge dette selv om man bestemmer seg for å endre størrelsen på resten av teksten.